docs/model-dev-guide/api-guides/apis-howto/api-keras-ug.rst

```diff
@@ -7,129 +7,155 @@
+of that to your ``main()`` function (or equivalent) in your training script:
 
-If using ``tf.data.Dataset``, users are required to wrap both their training and validation dataset
-using :meth:`self.context.wrap_dataset <determined.keras.TFKerasTrialContext.wrap_dataset>`. This
-wrapper is used to shard the dataset for distributed training. For optimal performance, users should
-wrap a dataset immediately after creating it.
+.. code:: python
 
-.. include:: ../../../_shared/note-dtrain-learn-more.txt
+   if __name__ == "__main__":
+       distributed, strategy = det.core.DistributedContext.from_tf_config()
+       with det.core.init(distributed=distributed) as core_context:
+           main(core_context, strategy)
 
-******************
- Define the Model
-******************
+*****************
+ Build the Model
+*****************
 
-Users are required wrap their model prior to compiling it using :meth:`self.context.wrap_model
-<determined.keras.TFKerasTrialContext.wrap_model>`. This is typically done inside
-:meth:`~determined.keras.TFKerasTrial.build_model`.
+Building a distributed-capable model is easy in keras; you just need to wrap your model building and
+compiling in the ``strategy.scope()``.  See the `TensorFlow documentation
+<https://www.tensorflow.org/tutorials/distribute/keras
+#create_the_model_and_instantiate_the_optimizer>`__ for more detail.
 
-******************************************
- Customize Calling Model Fitting Function
-******************************************
+.. code:: python
 
-The :class:`~determined.keras.TFKerasTrial` interface allows the user to configure how ``model.fit``
-is called by calling :meth:`self.context.configure_fit()
-<determined.keras.TFKerasTrialContext.configure_fit>`.
+   def main(core_context, strategy):
+       with strategy.scope():
+           model = my_build_model()
+           model.compile(...)
 
-***************
- Checkpointing
-***************
+***********************************
+ Create the ``DeterminedCallback``
+***********************************
 
-A checkpoint includes the model definition (Python source code), experiment configuration file,
-network architecture, and the values of the model's parameters (i.e., weights) and hyperparameters.
-When using a stateful optimizer during training, checkpoints will also include the state of the
-optimizer (i.e., learning rate). You can also embed arbitrary metadata in checkpoints via a
-:ref:`Python SDK <store-checkpoint-metadata>`.
+The :class:`~determined.keras.DeterminedCallback` will automatically integrate your training with
+the Determined cluster.  It reports train and test metrics, it reports progress, it saves
+checkpoints, and it uploads them to checkpoint storage.  It also handles preemption signals from the
+Determined master (such as if you pause your experiment), shutting down training, then it restores
+training from where it left off when the experiment continues.
```

Comment:
The :class:`~determined.keras.DeterminedCallback` automatically integrates your training with the Determined cluster. It reports both train and test metrics, reports progress, saves checkpoints, and uploads them to checkpoint storage. Additionally, it manages preemption signals from the Determined master (for example, when you pause your experiment), gracefully halting training and resuming from where it left off.

Reply:
I'm going to say:

```
gracefully halting training and *later resuming from where it left off
```

Because otherwise it sounds like, "wait why are you stopping and resuming as a result of a preemption signal?  Isn't the right action to just stop"?  The word "later" serves the purpose of my original "when the experiment continues" to indicate that there is some sort of trigger in between shutting down training and resuming.

Reply:
makes sense

Reply:
now there are redundant paragraphs

Reply:
doh.